components/logins/src/schema.rs

```diff
@@ -88,14 +88,19 @@
 //!    JSON.
 //!
 
+use crate::db::MigrationMetrics;
+use crate::encryption::EncryptorDecryptor;
 use crate::error::*;
 use lazy_static::lazy_static;
-use rusqlite::Connection;
+use rusqlite::{Connection, NO_PARAMS};
 use sql_support::ConnExt;
+use std::time::Instant;
 
 /// Note that firefox-ios is currently on version 3. Version 4 is this version,
 /// which adds a metadata table and changes timestamps to be in milliseconds
-pub const VERSION: i64 = 4;
+pub const VERSION: i64 = 5;
```

Comment:
I don't quite get why this isn't being reset? The comment above probably also needs updating (and I hope iOS isn't on version 3 still!?)

Reply:
To me, it feels more natural to think of the schema version as separate whether the database was using sqlcipher encryption.  So "version 4" specifies a particular schema, rather than also needing to say if it was plaintext or sqlcipher DB.  But I'm happy to change this if others like the resetting the numbers better.

Reply:
I don't feel too strongly either, but having it at "version 4" implies it's possible we will find a "version 3" out there. A comment was also added to explain why there are no migrations for numbers lower than this, because otherwise it would seem wrong.  IOW, I left this more as a note-to-self to say "hrm - maybe I don't actually understand the migration and what's going on here, I better come back and look more closely" - which is exactly the red-flag I'm thinking it might be worth avoiding

Reply:
I think I'm missing what the red-flag is, but resetting the schema sounds good to me.  I just pushed some code that does that.  in fact, I realized that the `sqlcipher_export` function doesn't export PRAGMA values, so we should be manually setting a version number anyways.  Are there any other PRAGMA values that we should be copying over?